### PR TITLE
Fix 'Build failed on Xcode, m1 Mac: Undefined symbol: getShaderBasePath()' #1046

### LINKED ISSUE #1046: Build failed on Xcode, m1 Mac: Undefined symbol: getShaderBasePath()
My Device: M1 Max MBP
I used the command Cmake -G "Xcode" to build the project, but I got the following messages 
-- Generating project file for example in /Users/neil/Documents/Vulkan/examples/texture3d
-- Could NOT find OpenMP_C (MISSING: OpenMP_omp_LIBRARY)
-- Could NOT find OpenMP_CXX (missing: OpenMP_omp_LIBRARY)
-- Could NOT find OpenMP (missing: OpenMP_C_FOUND OpenMP_CXX_FOUND)
-- Generating project file for example in /Users/neil/Documents/Vulkan/examples/texturearray
-- Generating project file for example in /Users/neil/Documents/Vulkan/examples/texturecubemap
-- Generating project file for example in /Users/neil/Documents/Vulkan/examples/texturecubemaparray

Then I opened the project in Xcode and got the following errors when building:
Undefined symbol: getShaderBasePath()
Linker command failed with exit code 1 (use -v to see invocation)

How do I fix the above problems?

### PR DESCRIPTION
**Pull request by Gadgeteering**
Fix for 'Build failed on Xcode, m1 Mac: Undefined symbol: getShaderBasePath()' 

Fixes #1046

### CODE REVIEW

**Review by SaschaWillems — CHANGES_REQUESTED** [review]
Thank you very much for this PR!

One note though: This PR adds a new vulkanExamples.xcodeproj in the root folder, while the repo already has an xcode folder for the MacOS related files. Any reason to have to different folders? Is the xcode folder outdated? If so, can it be removed?

Having to Mac related folders may be misleading.

**Comment by Gadgeteering** [comment]
The vulkanExamples.xcodeproj is the Mac Project and can be removed as this is generated by the Cmake

The examples.xcodeproj is an IOS & Mac project from the original examples from Brenwill Workshop

File: xcode/examples.xcodeproj/project.pbxproj
Comment (SaschaWillems) [inline xcode/examples.xcodeproj/project.pbxproj:632]: I don't know much about Mac developement, so before merging this: Is this `DEVELOPMENT_TEAM` team safe to be part of a public repository, or is this something security related that shouldn't be publicly visible?

Comment (Gadgeteering) [reply xcode/examples.xcodeproj/project.pbxproj]: I have removed the Xcode directory as it is no longer used to build the Vulkanexamples.

This has removed the above apple development team references.

When you build a application for iOS or MacOS apple require you to sign them with a a certificate generated by Xcode, if you are testing them you can use a developers certificate which is free. If you want to publish a application you need to setup a developers account and pay the annual fee.

If somebody wants to use the Vulcan examples they will need to build them on their MacBook and sign them to use them on their own MacBook only. (Unless the have a valid developers account)

Comment (SaschaWillems) [reply xcode/examples.xcodeproj/project.pbxproj]: Thanks for the clarification. I recently did some renaming and had to update a mac file. If you resolve the open merge conflict I'll merge this asap.

Comment (Gadgeteering) [reply xcode/examples.xcodeproj/project.pbxproj]: I am unable to resolve the conflicts as I do not have 'write access'

Comment (SaschaWillems) [reply xcode/examples.xcodeproj/project.pbxproj]: That's odd. You should be able to resolve the conflict locally, by merging master (from my repo) into your branch.

**Comment by Gadgeteering** [comment]
I have managed to resolve the conflicts

**Pull request merged**
